sdk/storage/azure-storage-files-datalake/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Breaking Changes
 
 - Removed `Azure::Storage::Files::DataLake::PackageVersion`.
+- Renamed `GetUserDelegationKeyOptions::startsOn` to `GetUserDelegationKeyOptions::StartsOn`.
```

Comment:
The same struct of options is used for both packages?

Reply:
yep